src/main/kotlin/com/expedia/graphql/schema/extensions/annotationExtensions.kt

```diff
@@ -23,73 +15,18 @@ internal fun KAnnotatedElement.getDeprecationReason(): String? = this.findAnnota
 
 internal fun Field.getDeprecationReason(): String? = this.getDeclaredAnnotation(Deprecated::class.java)?.getReason()
 
-internal fun Deprecated.getReason(): String? {
+private fun Deprecated.getReason(): String? {
     val builder = StringBuilder()
     builder.append(this.message)
-    if (!this.replaceWith.expression.isBlank()) {
+
+    if (this.replaceWith.expression.isBlank().not()) {
         builder.append(", replace with ")
         builder.append(this.replaceWith.expression)
     }
+
     return builder.toString()
 }
 
 internal fun KAnnotatedElement.isGraphQLIgnored() = this.findAnnotation<GraphQLIgnore>() != null
 
 internal fun KAnnotatedElement.isGraphQLID() = this.findAnnotation<GraphQLID>() != null
-
-private fun Annotation.getDirectiveInfo(): DirectiveInfo? {
-    return this.annotationClass.annotations
-        .filterIsInstance(DirectiveAnnotation::class.java)
-        .map { DirectiveInfo(this, it) }
-        .firstOrNull()
-}
-
-internal fun KAnnotatedElement.directives(generator: SchemaGenerator) =
-    this.annotations.asSequence()
-        .mapNotNull { it.getDirectiveInfo() }
-        .map { it.getGraphQLDirective(generator) }
-        .toList()
-
-internal fun KParameter.directives(generator: SchemaGenerator) =
```

Comment:
When moving, I deleted the `KParameter.directives()` since `KParameter` extends from `KAnnotatedElement` so it when can just use that extension function